mobile/package.json

```diff
@@ -36,29 +35,52 @@
     "@burstjs/http": "^0.4.0",
     "@burstjs/util": "^0.4.0",
     "@react-native-community/slider": "^2.0.0",
+    "@tradle/react-native-http": "^2.0.1",
     "@types/d3-shape": "^1.3.1",
     "@types/react-native-svg-charts": "^5.0.2",
+    "browserify-zlib": "^0.1.4",
+    "buffer": "^4.9.1",
     "d3-shape": "^1.3.5",
+    "dns.js": "^1.0.1",
+    "events": "^1.1.1",
+    "https-browserify": "0.0.1",
     "i18n-js": "^3.2.2",
     "lodash": "^4.17.13",
+    "namicorn": "^0.2.38",
+    "path-browserify": "0.0.0",
+    "process": "^0.11.10",
+    "punycode": "^1.4.1",
     "react": "16.8.3",
     "react-native": "^0.59.8",
-    "react-native-camera": "^3.6.0",
+    "react-native-camera": "^3.7.1",
+    "react-native-crypto": "^2.2.0",
     "react-native-gesture-handler": "^1.2.1",
     "react-native-keychain": "^3.1.3",
+    "react-native-level-fs": "^3.0.1",
     "react-native-localize": "^1.1.2",
     "react-native-markdown-renderer": "^3.2.8",
+    "react-native-os": "^1.2.5",
     "react-native-picker-select": "^6.3.2",
     "react-native-qrcode-scanner": "^1.2.3",
+    "react-native-randombytes": "^3.5.3",
     "react-native-svg": "^9.8.3",
     "react-native-svg-charts": "^5.3.0",
     "react-native-swipeout": "^2.3.6",
+    "react-native-tcp": "^3.3.2",
     "react-native-touch-id": "^4.4.1",
+    "react-native-udp": "^2.6.1",
     "react-native-version-number": "^0.3.6",
     "react-navigation": "^3.11.0",
     "react-redux": "^7.0.3",
+    "readable-stream": "^1.0.33",
     "redux": "^4.0.1",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "stream-browserify": "^1.0.0",
+    "string_decoder": "^0.10.31",
+    "timers-browserify": "^1.4.2",
+    "url": "^0.10.3",
+    "util": "^0.10.4",
+    "vm-browserify": "0.0.4"
```

Comment:
This is what I had to do to get node modules like `crypto`, `os`, etc working in react native. Gross.